openshift/templates/jobs/minio-backup/Dockerfile

```diff
@@ -1,15 +0,0 @@
-FROM alpine:latest
```

Comment:
Might be nice to have the restore pod available in case we ever need to access the old files again

Reply:
They have already been moved off and I think this would just create cruft - they should be safe on S3 appliance.

Reply:
Just there was a case recently where @BcGovNeal used this pod to run restic commands on the old backups to retrieve some lost files. I agree that any new data should be  handled externally, but in the case that we need to access the old backups again  it would be nice to have this for reference at least. Maybe make a note in the readme that  this is  for legacy data recovery only?

Reply:
Fixed in 95be983